sig-docs/charter.md

```diff
@@ -0,0 +1,54 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs is responsible for Kubernetes documentation.
+
+The responsibility for feature documentation lies with feature developers. SIG Docs sets standards for feature documentation, provides clear paths for docs contribution, and coordinates documentation updates during quarterly releases.
```

Comment:
What is SIG docs relationship with Kubernetes components outside of kubernetes/kubernetes?  What about other subprojects?

Reply:
> What is SIG docs relationship with Kubernetes components outside of kubernetes/kubernetes? What about other subprojects?

We may advise subprojects or other components outside of k/k; however, we are not responsible for documentation outside of the k/website repo.

Reply:
Can we update the charter to say that specifically.  E.g. sig-docs is responsible for the documentation on the kubernetes website and for components living in kubernetes/kubernetes.  sig-docs may be responsible for components living outside kubernetes/kubernetes if it is important to have the documentation on the k8s.io website.  sig-docs may consult on documentation for components outside k/k that exists outside of k8s.io, but is not obligated to.

Reply:
@pwittrock 👋 I have several questions about this statement:

> sig-docs may be responsible for components living outside kubernetes/kubernetes if it is important to have the documentation on the k8s.io website.

Who decides what is important?
What does "responsible" mean in this context?
Are there specific examples of products outside of k/k that need documentation on k8s.io? 
How does SIG Docs discover when a product needs docs to live on k8s.io?

There's a lot of unknowns there, many of which seem to obscure responsibility rather than clarify it. I recommend replacing this line with:

> SIG Docs is sometimes advises on Kubernetes components outside of the Kubernetes/Kubernetes GitHub repository, or on other Kubernetes subprojects, but is not responsible for the documentation of any of these components or projects.